imprint.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./src/output.css" />
+  <script>
+    const loadComponents = (id, url) => {
+      fetch(url)
+        .then((response) => response.text())
+        .then((html) => {
+          document.getElementById(id).innerHTML = html;
+        });
+    };
+
+    window.onload = () => {
+      loadComponents("greenNav", "src/sections/greenNav.html");
+      };
+      </script>
+</head>
+<body>
+  <div id="greenNav"></div>
+  <div
+  class="grid text-2xl ml-8 gap-20 font-body text-primaryGreen md:grid-cols-2 md:gap-32 lg:ml-28"
+>
+  <div class="flex flex-col gap-10 w-80 md:w-[42.81rem] md:gap-20 lg:gap-24">
+    <div>
+      <h4 class="font-bold pb-4 md:hidden md:text-xl lg:text-2xl lg:block">
+        IMPRINT
+      </h4>
+      <h4
+        class="font-bold pb-4 hidden md:text-xl md:block lg:text-2xl lg:hidden"
+      >
+        IMPRESSUM
+      </h4>
+      <h2 class="font-bold md:mb-3 md:text-3xl lg:text-5xl">
+        Agentur Baumeister
+      </h2>
+      <p class="md:mb-3 md:text-3xl lg:text-5xl">Katrein Baumeister</p>
+      <p class="md:mb-3 md:text-3xl lg:text-5xl">Böckhstraße 13</p>
+      <p class="md:mb-3 md:text-3xl lg:text-5xl">10967 Berlin</p>
+    </div>
+    <div class="text-2xl flex flex-col w-[21rem] md:w-[42.81rem] lg:text-5xl">
+      <p>+49 (0)30 - 490 827 87</p>
+      <p>hallo@agentur-baumeister.com</p>
+    </div>
+    <div class="md:text-3xl lg:text-5xl">
+      <h2 class="font-bold">UST-IDNR.</h2>
+      <p>DE 274 486 923</p>
+    </div>
+    <div>
+      <h2 class="font-bold md:text-3xl lg:text-5xl">REGISTERED OFFICE</h2>
+      <p class="hidden md:text-3xl lg:text-5xl lg:block">Berlin, Deutschland</p>
+      <p class="md:text-3xl lg:text-5xl lg:hidden">Berlin, Germany</p>
+    </div>
+  </div>
+  <div
+    class="flex flex-col md:pr-8 md:ml-20 md:justify-end md:items-end md:text-end md:gap-6 md:text-xl lg:pr-20 lg:gap-9 lg:text-2xl"
+  >
+    <h2 class="font-bold">DISCLAIMER</h2>
+    <p class="w-[22.19rem]">
+      Despite all due care, no liability can be accepted for the accuracy,
+      completeness and up to-dateness of the information provided. The same
+      applies to third-party websites to which www.agentur-baumeister.com refers
+      by means of hyperlinks. Katrein Baumeister, Agentur Baumeister accepts no
+      liability for the content of these websites.
+    </p>
+  </div>
+</div>
+<div class="flex justify-center pb-10 px-9 items-center sticky top-0 py-8">
+  <img class="w-96" src="/img/red.webp" alt="The red vector">
+</div>
```

Comment:
I want this to be in section folder, create a file, and then render it here the way you are accessing  green nav